test/e2e/e2e_test.go

```diff
@@ -170,12 +170,12 @@ type options struct {
 	IsRunningInCI        bool
 	ArtifactDir          string
 
-	defaultClusterOptions cmdcluster.Options
+	defaultClusterOptions core.CreateOptions
 	additionalTags        stringSliceVar
 }
 
-func (o *options) DefaultClusterOptions() cmdcluster.Options {
-	return o.defaultClusterOptions
+func (o *options) DefaultClusterOptions() *core.CreateOptions {
+	return &o.defaultClusterOptions
```

Comment:
This was intentionally returning a value type so that callers get a copy instead to avoid unintentional sharing of mutable state